.travis.yml

```diff
@@ -4,14 +4,15 @@ language: node_js
 matrix:
   include:
 
-    # Run linter once
+    # Run linter only, just once
     - node_js: '6'
       env: LINT=true
 
     # Run tests
-    - node_js: '4'
-    - node_js: '5'
+    - node_js: 'node'
```

Comment:
Yep, you raise an interesting point. As per our [open source support policies](https://github.com/springernature/frontend-playbook/blob/master/practices/open-source-support.md#node-versions) we'll only officially support LTS versions (so 4, 6, 8...).

Obviously that doesn't mean that we can't add node 7 to the test matrix - but we're trying to avoid the situation where we currently are, where in order to remove node 5 (which is unsupported) from the test matrix, we'll need to release a breaking change. I think that we prefer to not add odd versions of node at all.

Some of our colleagues suggested that it should be fine to add node 7 to the test matrix as long as we don't advertise it as _supported_, so it can removed at any time without further notice. So I think that adding 'node' instead of '7' is the best compromise as:
* It will run tests on version 7 right now.
* It will "automatically" remove 7 and add 8, the moment that version 8 is available on travis, without our intervention.
* We'll only need to modify the travis config again in order to add node 8 the moment that node 9 is to be released.

This helps us reduce the amount of time that we spend doing maintenance on our projects. As you can see, even if we're getting better at it, many of them are still outdated and are still running tests on node 5, and a few others are not even running tests on node 6 at all! 😱 

Adding 'node' to the travis.yml means that hopefully we won't have to modify that file until October, when v9 will be cut.

Hope it makes sense and thanks for the review!